import { CustomError } from "./CustomError";

export class RecipeNotFound extends CustomError {
    constructor() {
        super ('Receita n√£o encontrada', 404)
    }
}